Description: Don't use web version of mathjax; privacy leak
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2025-12-06
Forwarded: not-needed

--- a/docs/html/man-openmpi/man3/MPI_Init_thread.3.html
+++ b/docs/html/man-openmpi/man3/MPI_Init_thread.3.html
@@ -18,7 +18,7 @@
         <script src="../../_static/_sphinx_javascript_frameworks_compat.js"></script>
         <script src="../../_static/doctools.js"></script>
         <script src="../../_static/sphinx_highlight.js"></script>
-        <script async="async" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+        <script async="async" src="/usr/share/nodejs/mathjax-full/es5/tex-mml-chtml.js"></script>
     <script src="../../_static/js/theme.js"></script>
     <link rel="index" title="Index" href="../../genindex.html" />
     <link rel="search" title="Search" href="../../search.html" />
@@ -869,4 +869,4 @@
   </script> 
 
 </body>
-</html>
\ No newline at end of file
+</html>
